templates/flashes_helper.rb

```diff
@@ -0,0 +1,5 @@
+module FlashesHelper
+  def user_facing_flashes
+    flash.to_hash.slice("alert", "error", "notice", "success")
```

Comment:
I may be misunderstaing, but I just tried the following in an app:

* Back to `flash.each` in the `_flashes` partial.
* `add_flash_types :track_event` in the controller.
* `redirect_to edit_team_path(@team), track_event: "Created Team"` in the controller
* "Created Team" displays to the user, which it shouldn't, as I'm using the flash only for non-user facing event tracking

Is that what you were suggesting, or a different implementation?